jkube-kit/watcher/standard/src/test/java/org/eclipse/jkube/watcher/standard/DockerImageWatcherRestartContainerTest.java

```diff
@@ -162,7 +162,7 @@ private void assertPodTemplateSpecContainsImage(PodTemplateSpec podTemplateSpec,
     assertThat(podTemplateSpec)
         .extracting(PodTemplateSpec::getSpec)
         .extracting(PodSpec::getContainers)
-        .asList()
+        .asInstanceOf(InstanceOfAssertFactories.list(Container.class))
```

Comment:
With this, now you should be able to remove type call in `first()`

Reply:
@rohanKanojia, thanks for pointing this out! Fixed it.